RULES.md

```diff
@@ -695,6 +695,116 @@ Unimplemented rules from the AWS PCI DSS 3.2.1 Conformance Pack.
+| NZISM3.6-SageMakerNotebookInstanceKMSKeyConfigured      | The SageMaker notebook is not encrypted with a KMS key.                                                                                                                       | Because sensitive data can exist at rest in SageMaker notebook, enable encryption at rest to help protect that data                                                                                                                                                                                                                                          | 22.1.24.C.04[CID:4839]                                                                                   |
+| NZISM3.6-SecretsManagerUsingKMSKey                      | The secret is not encrypted with a KMS Customer managed key.                                                                                                                  | To help protect data at rest, ensure encryption with AWS Key Management Service (AWS KMS) is enabled for AWS Secrets Manager secrets. Because sensitive data can exist at rest in Secrets Manager secrets, enable encryption at rest to help protect that data                                                                                               | 22.1.24.C.04[CID:4839]                                                                                   |
+| NZISM3.6-SNSEncryptedKMS                                | The SNS topic does not have KMS encryption enabled.                                                                                                                           | To help protect data at rest, ensure that your Amazon Simple Notification Service (Amazon SNS) topics require encryption using AWS Key Management Service (AWS KMS) Because sensitive data can exist at rest in published messages, enable encryption at rest to help protect that data                                                                      | 22.1.24.C.04[CID:4839]                                                                                   |
+| NZISM3.6-EC2SecurityGroupOnlyTcp443                     | Only port tcp 443 shoudl be permitted in ingress security group.                                                                                                              | Not allowing ingress (or remote) traffic to ports other than tcp port 443 helps improve security                                                                                                                                                                                                                                                             | 18.1.13.C.02[CID:3205]                                                                                   |
+
+### Excluded Rules
```

Comment:
We should call out which controls are `SHOULD` and `MUST` (similar to the comment above) some of them do here and some of them don't